x-pack/test_serverless/functional/config.base.ts

```diff
@@ -116,6 +116,7 @@ export function createTestConfig(options: CreateTestConfigOptions) {
         integrations: {
           pathname: '/app/integrations',
         },
+        ...(options.apps ?? {}),
```

Comment:
@TattdCodeMonkey I don't see any change in Search projects configs where custom `apps` are passed. Do I miss something? 
Tbh I don't see an issue to add new paths directly in base config

Reply:
I added them [search/config.ts](https://github.com/elastic/kibana/blob/6ad85254aed79b3eea52b83950375fffbb08ddd6/x-pack/test_serverless/functional/test_suites/search/config.ts) & [search/config.feature_flags.ts](https://github.com/elastic/kibana/blob/6ad85254aed79b3eea52b83950375fffbb08ddd6/x-pack/test_serverless/functional/test_suites/search/config.feature_flags.ts)

This allows to register apps that are only used in search, but more importantly in this case one of the app paths is updated when the FF is enabled which is why I needed to add this. I wanted to ensure the test continue to pass in both cases.